src/pudl/metadata/fields.py

```diff
@@ -2241,6 +2249,31 @@
         "type": "boolean",
         "description": "Indicates whether the generator is part of a solid fuel gasification system",
     },
+    "stack_id_eia": {
+        "type": "string",
+        "description": (
+            "The value EIA uses to identify a stack or chimney on a power plant where "
+            "emissions from the combustion process are released into the atmosphere."
+        ),
+    },
+    "stack_flue_id_eia": {
+        "type": "string",
+        "description": (
+            "The value EIA uses to identify hybrid stack and flue units. Plants with "
+            "individual stack and flue IDs do not have a stack_flue_id value."
+        ),
+    },
+    "stack_flue_id_pudl": {
+        "type": "string",
+        "description": (
+            "The stack_id_eia value and the flue_id_eia value seperated by an "
```

Comment:
I would suggest something like this: 
"A unique identifier of the stack and/or flue unit, produced by PUDL. If there is an ID for a combined stack-flue unit (stack_flue_id_eia), this value is used. Otherwise, this value is a combination of the EIA stack ID (stack_id_eia) and the flue ID (flue_id_eia), separated by an underscore. If there is no flue ID, this is equivalent to the stack ID provided by EIA. This value is used as the primary key for the boiler_stack_flue_assn_eia860 table and shouldn't necessarily be used to connect the values in this table to those of another."

With regards to the last line, do we not want to add this column to the other cooling and stack flue tables to enable joining by this column?

Reply:
We certainly could, I just want to preserve/emphasize the fact that the relationship between stacks and flues is not necessarily 1:1.

Reply:
Also I don't think it's necessarily a unique identifier either--just when paired with the `report_date` and `plant_id_eia`